README.md

```diff
@@ -3,3 +3,7 @@
 Connector API is a general API for communication between Mews and external applications or systems. The API is typically consumed by other cloud services that work with data of hotels in Mews \(e.g. revenue management systems, cloud POS systems\). It can also be used by applications that are running on site in the hotel and mediate communication between Mews and local devices \(e.g. POS systems, printers and other physical devices, kiosks etc\).
 
 To see scenarios where this API might be used, have a look at the [Use cases](use-cases/README.md) that may guide you through the implementation. Before the actual implementation, get familiar with the [Guidelines](guidelines/) which describe how to communicate with the API. All operations and functions supported by the API are described in [Operations](operations/), [Websockets](websockets.md) and [Webhooks](webhooks.md) sections. If you are interested in changes and updates of this API, check the [Changelog](changelog.md) or follow the Connector API [GitHub repository](https://github.com/MewsSystems/gitbook-connector-api/tree/master).
+
+---
+>As an alternative to using this website, you can find all the documentation pages [here on GitHub](https://github.com/MewsSystems/gitbook-connector-api/tree/master).
```

Comment:
... `this website`... it does not make sense to me, it's confusing for users who are already on GitHub. Why don't we write something like this: 

if you are experiencing an issue using the GitBook platform, you can use our default documentation repository on [Github] (...)

Reply:
+ also the place does not make sense to me :) This is the shortest page of our DOC. This page is loaded correctly. I would say we should put it on the top of each page.

Reply:
Yes of course, 'this website' doesn't make sense as the reader could be seeing it on GitHub or GitBook. It's also a good point about placement, although the home page is the only place you can be guaranteed to be able to read it ok :-). I'm not sure how much we want to draw attention to issues with GitBook - can we discuss it in our meeting today?